pladder/bjbot.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pladder.plugin import Plugin
+
+class BjBotPlugin(Plugin):
+    def __init__(self, bot):
+        super().__init__()
+        self._datorbas = dict()
+        self.importera(self._datorbas, os.path.join(bot.statedir, "bunny.txt"))
+        if not bot is None:
+            bot.register_command("jb", self.jb)
+
+    def importera(self, db, path):
+        if os.path.exists(path):
+            with open(path) as fp:
```

Comment:
For this to work correctly, we need an "encoding" keyword argument to open() that matches the encoding of the file itself. The file seems to be Latin-1. Should we change the file or the code?